src/otx/core/utils/build.py

```diff
@@ -16,14 +16,20 @@
     from omegaconf import DictConfig
     from torch import nn
 
-def build_mm_model(config: DictConfig, model_registry: Registry, load_from: str) -> nn.Module:
+def build_mm_model(config: DictConfig,
+                   model_registry: Registry,
+                   load_from: str,
+                   load_backbone_only: bool = False) -> nn.Module:
     """Build a model by using the registry."""
     try:
         model = model_registry.build(convert_conf_to_mmconfig_dict(config, to="tuple"))
     except AssertionError:
         model = model_registry.build(convert_conf_to_mmconfig_dict(config, to="list"))
+    model_device = next(model.parameters()).device
```

Comment:
Do we need to do? AFAIK, Pytorchlightening automatically use the accelerator even though we load the model checkpoint as CPU

Reply:
I have an issue with that. When I launched training on CPU -> device-side assert triggered. I will reinstall env and try to reproduce this issue

Reply:
deleted [29dd2eb](https://github.com/openvinotoolkit/training_extensions/pull/2693/commits/29dd2eb8810801d217caef839453df331d47e086)